### ISSUE TITLE
Some actions are missing in accessibility menu

### ISSUE THREAD

**Issue by charlag** [tuskyapp/Tusky#1710]
- bookmark
- copy link
- download media
- block
- mute
- report

* * * *
- Tusky Version: 10.0 beta 1-aba36ca
- Android Version: 10
- Android Device: Fairphone 3

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
